.travis.yml

```diff
@@ -7,6 +7,23 @@ matrix:
   include:
   - jdk: oraclejdk7
     script: mvn -q deploy --settings .travis-settings.xml -Dno.gem.deploy=true
+  - jdk: oraclejdk7
+    env: ANDROID_EXAMPLES=true
+    language: android
+    android:
+      components:
+        - build-tools-21.1.1
+        - platform-tools
+        - extra-android-support
+        - android-21
+        - sys-img-armeabi-v7a-android-21
+    script: mvn install -P android-examples -pl examples/android,examples/android/android-test -am -amd -Dno.gem.deploy=true -Dandroid.device=test
+    install: mvn install -P android-examples -pl examples/android,examples/android/android-test -am -amd -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
Most often maven would detect that the build artefacts is already up to date the second time and not rebuild, so the idea is probably that it should not add so much time to run maven once (without tests) before the actual script phase. For instance [this build](https://travis-ci.org/cucumber/cucumber-jvm/jobs/46895638), where test are not run neither the first nor second time, maven list the build time for cucumber-core as 49s the first time, and 3s the second time. In case of building Android module with the android-maven-plugin, it looks as if everything is rebuild the second time, and then you definitely pay for a lot more execution time.